src/config-serializer/config-items/macro-action/EditableMacroAction.ts

```diff
@@ -0,0 +1,205 @@
+import {MacroAction, macroActionType} from './MacroAction';
+import {values as _values } from 'lodash';
+
+import {KeyAction, keyActionType} from '../key-action/KeyAction';
+import {KeystrokeAction} from '../key-action/KeystrokeAction';
+import {KeyMacroAction} from './KeyMacroAction';
+import {DelayMacroAction} from './DelayMacroAction';
+import {MouseButtonMacroAction} from './MouseButtonMacroAction';
+import {MoveMouseMacroAction} from './MoveMouseMacroAction';
+import {ScrollMouseMacroAction} from './ScrollMouseMacroAction';
+import {TextMacroAction} from './TextMacroAction';
+
+const macroActionTypeValues = _values(macroActionType);
```

Comment:
Provided that nobody tries to use it on Safari (or the UI view version) or there is a polyfill for it. I used lodash because it avoids a fair bit of this kind of checking if an ES6 feature is implemented across browsers.